docs/data/base/components/switch/switch.md

```diff
@@ -0,0 +1,48 @@
+---
+product: base
+title: React Switch component
+components: SwitchUnstyled
+githubLabel: 'component: switch'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#switch
+packageName: '@mui/base'
+---
+
+# Switch
+
+<p class="description">Switches toggle the state of a single setting on or off.</p>
+
```

Comment:
Should we add the features list?!

Reply:
Not sure we should add it in all components. The Switch is a rather simple component, not sure it deserves a features list :)

Reply:
Hm, yeah... it's a valid point. Though I confess I like the consistency of having in all of them, even in simpler ones like the Switch. I appreciate the [Radix team](https://www.radix-ui.com/docs/primitives/components/switch) for doing that, it's easy to scan through!

Reply:
Well, from their list, I would expect all components to support those features of hey are input components :D

Reply:
Let's revisit this for all components and make sure we at least don't miss it on the more complex ones.